server/src/main/java/org/elasticsearch/index/mapper/MappedFieldType.java

```diff
@@ -200,7 +202,7 @@ public boolean isSearchable() {
      */
     public boolean isAggregatable() {
         try {
-            fielddataBuilder("");
+            fielddataBuilder("", () -> null);
```

Comment:
This bit deserves a TODO or, well, something. Because it ain't gonna work properly for anything that needs the context for real.

Reply:
this is only to see if the method throws error because the field is not aggregatable, when would the null lookup matter? Btw this method will most likely need to be overridden by the runtime fields mapped filed type to return true all the time.

Reply:
Yeah. I do think that runtime fields will just want to override this to `true`. Or something like that. I'm fine with the change as it stands.